pkg/p2p/peer/peer_impl.go

```diff
@@ -78,10 +81,12 @@ func (a *PeerImpl) SendMessage(m proto.Message) {
 		zap.S().Errorf("Failed to send message %T: %v", m, err)
 		return
 	}
+	zap.S().Named(logging.NetworkDataNamespace).Debugf("[%s] Sending to network: %s",
+		a.id.String(), base64.StdEncoding.EncodeToString(b))
```

Comment:
Use `logNetworkData` function.

Reply:
Removed the function. It was a bad idea because of single place to use and the necessity to pass a direction of transfer overcomplicates such a simple logic.